_i18n/en.yml

```diff
@@ -343,7 +343,7 @@ press-kit:
   whycommunity1: Monero is trusted for more private transactions than all other cryptocurrencies combined.
   whyconf: The Monero community has substantial involvement at Defcon, Grayhat and the CCC (Chaos Computer Club). The Monero Konferenco, an academic conference, is hosted annually.
   whyconf1: We encourage everyone to include community members from the largest privacy project in related news articles and events.
-  altpk-1: Picture of a woman standing in front of a 'Monero Talk' banner
+  altpk-1: Picture of a woman standing in front of a Monero Talk banner
```

Comment:
Well, single apostrophes instead of double qoutes do work, but I see how you would like to achieve consistency over all languages.

Reply:
It's not really about consistency. Weblate replaces single quotes with double quotes and people might not pay attention and use double quotes when they shouldn't, resulting in a broken page. Removing the quotes will avoid the problem.